hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1096,6 +1096,57 @@ def hpat_pandas_series_ge_impl(self, other):
     raise TypingError('{} The object must be a pandas.series and argument must be a number. Given: {} and other: {}'.format(_func_name, self, other))
 
 
+@overload_method(SeriesType, 'idxmin')
+def hpat_pandas_series_idxmin(self, axis=None, skipna=True):
```

Comment:
@1e-to I see in documentation the interface is a bit diffrent. It looks like `self, axis=0, skipna=True, *args` The last argument (`*args`)is not used but it might be good to put it in the function parameters

Reply:
Fixed